pipeline/tests/asm.rs

```diff
@@ -201,17 +203,26 @@ fn vm_to_block_array() {
 }
 
 #[test]
-fn vm_to_block_different_length() {
-    let f = "asm/vm_to_block_different_length.asm";
+fn dynamic_vadcop() {
+    let f = "asm/dynamic_vadcop.asm";
+
+    let mut pipeline_gl = make_simple_prepared_pipeline(f);
+    let witness = pipeline_gl.compute_witness().unwrap();
+    let witness_by_name = witness
+        .iter()
+        .map(|(k, v)| (k.as_str(), v))
+        .collect::<BTreeMap<_, _>>();
+
+    // Spot-check some witness columns to have the expected length.
+    assert_eq!(witness_by_name["main.X"].len(), 128);
```

Comment:
Note that that's because we only detect the loop at row 100. But I think that's fine, it's still a small number.